templates/partials/instance_description.rs.html

```diff
@@ -21,7 +21,7 @@ <h2>@i18n!(ctx.1, "About {0}"; instance.name)</h2>
                 @Html(instance.short_description_html)
                 <section class="stats">
                     <div>
-                        <p>@Html(i18n!(ctx.1, "Home to <em>{0}</em> people"; n_users))</p>
+                        <p>@Html(i18n!(ctx.1, "Home to <em>{0}</em> users"; n_users))</p>
```

Comment:
I would have preferred to use "people" for both, it feels more like they are real human beings than "users" IMO.

Reply:
Right. Done :-)